package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "survive-the-monsters",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^5.4.9"
```

Comment:
**praise**: Good choice of using Vite for development, which will enhance the development experience with its fast build times.